packages/@aws-cdk/core/lib/tag-manager.ts

```diff
@@ -259,6 +283,24 @@ export class TagManager {
     return this.tagFormatter.formatTags(Array.from(this.tags.values()));
   }
 
+  /**
+   * Creates the Tags property object
+   *
+   * The object contains a single key for the tag property name with a value of
+   * the properly formatted tags
+   */
+  public asCfnProperty(): {[key: string]: any} {
+    const tagsProp: {[key: string]: any} = {};
+    tagsProp[this.tagPropertyName] = this.renderTags();
+    return tagsProp;
+  }
```

Comment:
Why do we need this method? `renderTags` and `tagPropertyName` are both public. Why not just put this logic within `CfnResource`?

Reply:
So I started to consider consolidation but how should I view a breaking change? If I take away a public method is that breaking? Should we make the refactor and start deprecating methods.

Reply:
Yes, let's move to `CfnResource` and keep` tagPropertyName` and `renderTags` public.